mlrun/feature_store/api.py

```diff
@@ -327,6 +327,7 @@ def ingest(
                                     False for scheduled ingest - does not delete the target)
 
     """
+    user_return_df = return_df
```

Comment:
make sure its ok by Yaron and:
1. Add a :return: doc string that specify that it will return the DF if `return_df` is `True`.
2. Change the return type hint to `Union[pd.DataFrame, None]`